plugins/code-coverage-backend/README.md

```diff
@@ -2,6 +2,50 @@
 
 This is the backend part of the `code-coverage` plugin. It takes care of processing various coverage formats and standardizing them into a single json format, used by the frontend.
 
+## Installation
+
+`yarn workspace backend add @backstage/plugin-code-coverage-backend`
+
+Next you will need to add code to setup the plugins router, in `packages/backend/src/plugins/codecoverage.ts` add the following:
```

Comment:
Can you explicitly stage that this file needs to be created? I've often seen people confused by that part

Reply:
Sorry, I don't see any changes here. Basically something like this would work: "Next you will need to add code to setup the plugin's router. First create a `codecoverage.ts` file here: `packages/backend/src/plugins`. Now add the following as its content:"

Reply:
So use a diff here as well?